src/index.html

```diff
@@ -7,6 +7,423 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="wrapper">
+      <header class="header">
+        <div class="header__logo">
+          <a
+            href="#"
+            class="header__link"
+          >
+            <img
+              src="images/logo.png"
+              alt="#"
+              class="header__picture"
+             >
+          </a>
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a href="#" class="nav__link is-active">
+                  Apple
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Samsung
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Смартфоны и телефоны
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                  data-qa="nav-hover"
+                >
+                  Ноутбуки и компьютеры
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Гаджеты
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Планшеты
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Фото
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Видео
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <main class="main">
+        <div class="catalog">
+          <div class="catalog__container">
+            <div class="catalog__row">
+              <div class="card" data-qa="card">
```

Comment:
Better delete unnecessary wrappers and use mix here. Like in `header`.
More detail in [video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911).

And add changes to styles according to changes in html.
```suggestion
      <main class="catalog page__catalog">
              <div class="card catalog__card" data-qa="card">
```

Reply:
+